app/api/gatcg_spoilers/route.ts

```
@@ -0,0 +1,46 @@
+    dataContainer.data.push({
+      uuid: '',
+      name: '',
+      card_image_url: '',
+      back_card_name: '',
+      back_card_image_url: '',
+    });
+    dataContainer.data[i].uuid = crypto.randomUUID();
+    dataContainer.data[i].name = responseJson.spoilers[i].card_name;
+    dataContainer.data[i].card_image_url =
+      'https://cgs.games/api/proxy/alpha.silvie.gg' + responseJson.spoilers[i].card_image_url;
+    if (responseJson.spoilers[i].back_card && responseJson.spoilers[i].back_card.card_name) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)